packages/manager/modules/storage/src/storages.service.js

```diff
@@ -1,68 +1,74 @@
 export default class StoragesService {
   /* @ngInject */
-  constructor($q, OvhHttp) {
+  constructor($q, iceberg) {
     this.$q = $q;
-    this.OvhHttp = OvhHttp;
+    this.iceberg = iceberg;
   }
 
   getStorageNasha() {
-    return this.OvhHttp.get(`/dedicated/nasha/`, {
-      rootPath: 'apiv6',
-    }).then((pool) => {
-      return this.$q.all(
-        pool.map((item) =>
-          this.OvhHttp.get(`/dedicated/nasha/${item}`, {
-            rootPath: 'apiv6',
-          }).then((res) => {
-            return {
-              id: res.serviceName,
-              region: res.datacenter,
-              product: 'NasHA',
-            };
-          }),
-        ),
-      );
-    });
+    return this.iceberg(`/dedicated/nasha/`)
+      .query()
+      .execute()
+      .$promise.then(({ data: pool }) => {
+        return this.$q.all(
+          pool.map((item) =>
+            this.iceberg(`/dedicated/nasha/${item}`)
+              .query()
+              .execute()
+              .$promise.then(({ data: res }) => {
+                return {
+                  id: res.serviceName,
+                  region: res.datacenter,
+                  product: 'NasHA',
+                };
+              }),
+          ),
+        );
+      });
   }
 
   getStorageNas() {
-    return this.OvhHttp.get(`/dedicated/nas/`, {
-      rootPath: 'apiv6',
-    }).then((pool) => {
-      return this.$q.all(
-        pool.map((item) =>
-          this.OvhHttp.get(`/dedicated/nas/${item}`, {
-            rootPath: 'apiv6',
-          }).then((res) => {
-            return {
-              id: res.serviceName,
-              region: res.datacenter,
-              product: 'NasHA',
-            };
-          }),
-        ),
-      );
-    });
+    return this.iceberg(`/dedicated/nas/`)
+      .query()
+      .execute()
+      .$promise.then(({ data: pool }) => {
+        return this.$q.all(
+          pool.map((item) =>
+            this.iceberg(`/dedicated/nas/${item}`)
+              .query()
+              .execute()
+              .$promise.then(({ data: res }) => {
+                return {
+                  id: res.serviceName,
+                  region: res.datacenter,
+                  product: 'NasHA',
```

Comment:
consider it move this to constants file as you are using it more than once. It would be easy to refactor.

Reply:
As explain by private message, there is no need to create constant for this.